pydrive2/auth.py

```diff
@@ -197,9 +198,9 @@ def __init__(
         self.settings = settings or self.DEFAULT_SETTINGS
         ValidateSettings(self.settings)
 
-        self._storages = self._InitializeStoragesFromSettings()
-        # Only one (`file`) backend is supported now
-        self._default_storage = self._storages["file"]
+        storages, default = self._InitializeStoragesFromSettings()
+        self._storages = storages
+        self._default_storage = default
```

Comment:
Original names are a bit unwieldy, so useing temp short ones. Not the prettiest, but I'm trying to make minimal changes.